app/models/doi.rb

```diff
@@ -962,7 +962,7 @@ def self.query(query, options = {})
     minimum_should_match = 0
 
     filter << { terms: { doi: options[:ids].map(&:upcase) } } if options[:ids].present?
-    filter << { term: { resource_type_id: options[:resource_type_id].underscore.dasherize } } if options[:resource_type_id].present?
+    filter << { terms: { resource_type_id: options[:resource_type_id].split(",").map(&:underscore).map(&:dasherize) } } if options[:resource_type_id].present?
```

Comment:
Code changes look perfect, can we format it like this,

```
  if options[:resource_type_id].present?
    resource_type_ids = options[:resource_type_id]
                          .split(",")
                          .map { |id| id.strip.underscore.dasherize }
   filter << { terms: { resource_type_id: resource_type_ids } }
  end
  ```

Reply:
I've done this, thanks for the suggestion :)